config/config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
there is a need for separate config files pere environment: https://github.com/mojaloop/thirdparty-scheme-adapter/tree/master/config
you should also specify here ENV & SHOULD_USE_IN_MEMORY_DB missed props